src/app/catalog/art-collection-section/art-collection.service.ts

```diff
@@ -12,8 +12,10 @@ import { ArtCollection } from './art-collection.model';
 export class ArtCollectionService {
   constructor(private http: HttpClient) {}
 
-  getAllArtCollections(): Observable<ArtCollectionPreview[]> {
-    return this.http.get(`${environment.API_URL}/art-collection`).pipe(map(result => result as ArtCollectionPreview[]));
+  getAllArtCollections(filter?: string): Observable<ArtCollectionPreview[]> {
+    return this.http
+      .get(`${environment.API_URL}/art-collection?filter=${filter}`)
```

Comment:
If the filter is undefined, we should do any filtering. This happens for art collections, artists, and artwork. That functionality should be moved to another service.

Reply:
This will happen as part of a cleanup for lists